cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -942,4 +952,58 @@ private static String trimStartingWhitespaces(String str) {
     private static String removeLastCharacter(String str) {
         return str.substring(0, str.length() - 1);
     }
+
+    /**
+     * Check if files of the given template exist in a given path.
+     *
+     * @param template given string
+     * @param packagePath given path
+     * @throws URISyntaxException if URI syntax exception occurred
+     * @throws IOException if IO exception occurred
+     */
+    public static String checkTemplateFilesExists(String template, Path packagePath) throws URISyntaxException,
+            IOException {
+        Path templateDir = getTemplatePath().resolve(template);
+        Stream<Path> paths = Files.list(templateDir);
+        List<Path> templateFilePathList = paths.collect(Collectors.toList());
+        String existingFiles = "";
+        for (Path path : templateFilePathList) {
+            String fileName = path.getFileName().toString();
+            if (!fileName.endsWith(ProjectConstants.BLANG_SOURCE_EXT) && Files.exists(packagePath.resolve(fileName))) {
+                existingFiles += fileName + FILE_STRING_SEPARATOR;
```

Comment:
We can add a test with filename that does match with the given condition. wdyt?

Reply:
This is already covered in an existing test